src/library_webgl.js

```diff
@@ -2223,12 +2216,19 @@ var LibraryGL = {
   },
 
   glUniform1iv__sig: 'viii',
+#if GL_POOL_TEMP_BUFFERS && MIN_WEBGL_VERSION == 1
+  glUniform1iv__deps: ['_miniTempIntBuffers'],
+#endif
   glUniform1iv: function(location, count, value) {
 #if GL_ASSERTIONS
     GL.validateGLObjectID(GL.uniforms, location, 'glUniform1iv', 'location');
     assert((value & 3) == 0, 'Pointer to integer data passed to glUniform1iv must be aligned to four bytes!');
 #endif
 
+#if MIN_WEBGL_VERSION == 2
+    GLctx.uniform1iv(GL.uniforms[location], HEAP32, value>>2, count);
+#else
```

Comment:
this part of the change seems unrelated to the temp buffers, or am I missing something?

Reply:
Yeah, that is to optimize code size on WebGL 2 only code.